ryanbas21/ryan_blog
# Release (#30)

* Start (#2)

* init commit

* Codacy
adding codacy

* Quick fixes

* Updated SSR and Redux

* quick fix

* Delete tags.lock

* Delete tags.temp

* Prod fix (#7)

*  with '#' will be ignored, and an empty message aborts the commit.

* bug(rebase):

removing temp tag file

* prod(fix)
quick fix for prod build
updating gitignore and build for prod

* bug(fix contentful deletion)

quick udpate on missing deletion

didn't delete a contentful import

* feat(SSR)

Added SSR, removed test post

Added server side rendering, removed a test post, and implemented tags

* feat

fix gatsby browser

clean up gatsby browser file since I don't need it

Added typography

* bug(deply-ssr-bug) (#9)

fix ssr

* bug(ssr?)

SSR not working?

Check

* quick change

* SSR implementation for release (#8) (#10)

* Start (#2)

* init commit

* Codacy
adding codacy

* Quick fixes

* Updated SSR and Redux

* quick fix

* Delete tags.lock

* Delete tags.temp

* Prod fix (#7)

*  with '#' will be ignored, and an empty message aborts the commit.

* bug(rebase):

removing temp tag file

* prod(fix)
quick fix for prod build
updating gitignore and build for prod

* bug(fix contentful deletion)

quick udpate on missing deletion

didn't delete a contentful import

* feat(SSR)

Added SSR, removed test post

Added server side rendering, removed a test post, and implemented tags

* feat

fix gatsby browser

clean up gatsby browser file since I don't need it

Added typography

* bug(deply-ssr-bug) (#9)

fix ssr

* bug(ssr?)

SSR not working?

Check

* quick change

* quick change (#11)

* Update README.md

* feat(tags): Tags should now work (#13)

Added tags to feed and refactored view logic

* feat(tags): Tags should now work (#14)

Added tags to feed and refactored view logic

* feat(tags): Tags should now work (#15)

Added tags to feed and refactored view logic

* feat(tags): Tags should now work (#16)

Added tags to feed and refactored view logic

* Tags (#18)

* feat(tags): Tags should now work

Added tags to feed and refactored view logic

* bug(tag import)

Fixed tag import

Fixed the tag import

* remove node 6

* feat(view-tests): Adding View test (#17)

* feat(view-tests): Adding View test
ADded react-testing-library

React- testing - library tests

* remove node 6

* Update .travis.yml

* Update package.json (#20)

* Likes (#22)

* React Testing Library :woo!

added view tests
Not sure if I am doing view tests properly but I'm trying

* Update package.json

* Update package.json

* feat
Added a integration test for adding replies

Added one integration test that tests when a user adds text to the textbox and submits it, that it is rendered

* feat

Like Button

Like button should appear near the article to be clicked

* feat(test)

Updated the test to be more specific and better!

Actually tests that the heart turns red now.

* Update README.md

* Comments (#23)

* Update package.json

* feat
Added a integration test for adding replies

Added one integration test that tests when a user adds text to the textbox and submits it, that it is rendered

* updated code

* quick removal of default prop

* V2 (#26)

* This commit encapsulates most of the v2 upgrade.

Still need to fix the title component and update react-hooks and ramda.

There also exists quite a few type errors on this branch currently that will get resolved, but not
going to waste time with it when most of the code will get chopped away

* updated to properly ascend by date

* updated Link

* Updated v2 and have a new post!

* Adding a favicon (#28)

* Updating package
